dotnetv3/S3/ManageACLsExample/ManageACLs.cs

```diff
@@ -107,7 +107,7 @@ public static async Task<bool> CreateBucketWithCannedACLAsync(IAmazonS3 client,
         /// PutBucketAsync.</param>
         /// <param name="bucketName">The Amazon S3 bucket for which we want to get the
         /// ACL list.</param>
-        /// <returns>Returns an Amazon S3AccessCntrolList returned from the call to
+        /// <returns>Returns an S3AccessCntrolList returned from the call to
```

Comment:
S3AccessControlList (typo)

Reply:
Good grief! I looked at that I don't know how many times and didn't see it. Thanks!